bin/list_rc_log.py

```diff
@@ -0,0 +1,256 @@
+    #        if 'gregkh' not in efrom:
+            if 'linaro.org' in efrom:
+                print("%d: %s" % (x, efrom))
+                if 'replies' in fg[inrt]:
+                    fg[inrt]['replies'].append(msg)
+                else:
+                    fg[inrt]['replies'] = [msg]
+
+        x += 1
+
+    # print(str(from_greg))
+    return fg
+
+
+class Review:
+    request = None
+    reply = None
+    elapsed_time = None
+
+    def __init__(self, req, reply):
+        self.request = req
+        self.reply = reply
+
+    def set_request(self, req):
+        self.request = req
+
+    def set_reply(self, reply):
+        self.reply = reply
+
+    def calc_elapsed_time(self):
+        request_time = msg_get_dt(self.request)
+        if (self.reply):
+            reply_time = msg_get_dt(self.reply)
+        else:
+            reply_time = NOW
+
+        # This is a datetime.timedelta
+        diff = reply_time - request_time
+        self.elapsed_time = diff
+
+    def get_elapsed_time(self):
+        self.calc_elapsed_time()
+
+        days = self.elapsed_time.days
+        hours = days * 24
+        hours += int(self.elapsed_time.seconds / 3600)
+        minutes = int((self.elapsed_time.seconds % 3600) / 60)
+        return "%d:%02d" % (hours, minutes)
+
+    def get_sla_mark(self):
+        self.calc_elapsed_time()
+        et = self.elapsed_time
+        hours = et.seconds / 3600
+        if et.days >= 2:
+            return ">48h"
+        elif et.days < 2 and et.days >= 1:
+            return "<48h"
+        elif et.days < 1 and hours >= 8:
+            return "<24h"
+        else:
+            return "<8h"
+
+    def get_linux_version(self):
+        sub = self.request['subject']
+        if not sub.endswith("-stable review"):
+            return None
+        if not sub.startswith("["):
+            return None
+        sub1 = sub.replace('-stable review', '')
+        return sub1[sub1.rfind(' ')+1:]
+
+    def get_ymd(self):
+        dt = msg_get_dt(self.request)
+        return dt.strftime("%Y-%m-%d")
+
+    def get_from(self):
+        return self.reply['from']
+
+    def get_id(self):
+        return self.reply['message-id']
+
+    def get_regressions_detected(self):
+        mfrom = self.reply['from']
+        if 'linaro.org' not in mfrom:
+            return False
+
+        body = self.reply.get_payload().lower()
+        if 'no regressions on ' in body:
+            return False
+        else:
+            return True
+
+    def get_has_reply(self):
+        if self.reply:
+            return True
+        return False
+
+
+if __name__ == '__main__':
+    # Arguments
+    ap = argparse.ArgumentParser()
+    g = ap.add_mutually_exclusive_group(required=False)
+    g.add_argument(
+        "-d", "--days", help="Number of days back to look at", type=int, default=7)
+    # TODO: Support a parseable date
+    # g.add_argument("-s", "--since", help="Look as far as the given date")
+    args = vars(ap.parse_args())
```

Comment:
I would not convert it to vars. If you remove vars() you can use args.parameter syntax when checking for values. A few lines below you could use args.days instead of copying from array.